packages/components/src/components/input/input.styles.ts

```diff
@@ -102,6 +111,11 @@ export const styles: JssStyle = {
       },
     },
   },
+  'input--disable-resize': {
+    '& .input__textarea': {
+      resize: 'none',
```

Comment:
We need to expand this to allow for multiple settings, as resize could be: `unset | none | vertical | horizontal`

Maybe we could achieve it through the component and using the `jss` `stylesheet.addRules`?